src/components/timeline/TrackThread.js

```diff
@@ -107,6 +111,11 @@ class TimelineTrackThread extends PureComponent<Props> {
     });
   };
 
+  componentDidUpdate() {
+    const { threadIndex, height, reportTrackThreadHeight } = this.props;
+    reportTrackThreadHeight(threadIndex, height);
+  }
```

Comment:
I'm surprised that a track thread may change between renders. Shouldn't we report this once in `componentDidMount` ?
And in that case we don't need to guard for multiple dispatches in `reportTrackThreadHeight`.

Reply:
Hmmm..... this may not be needed, but I'd like to be a little bit defensive here. I'm going to keep it, but add more comments explaining it may not be needed.